sklearnex/decomposition/pca.py

```diff
@@ -17,6 +17,7 @@
 import logging
 
 from daal4py.sklearn._utils import daal_check_version
+from sklearnex._config import get_config

```

Comment:
minor comment - let's unify these eventually
```suggestion
from .._config import get_config

```